frontend/public/components/utils/workload-pause.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+// eslint-disable-next-line no-unused-vars
+import { K8sKind, k8sPatch, K8sResourceKind} from '../../module/k8s/index';
+import { errorModal } from '../modals/index';
+
+export const togglePaused = (model: K8sKind, obj: K8sResourceKind) => {
+  const patch = [{
+    path: '/spec/paused',
+    op: 'replace',
+    value: !obj.spec.paused,
+  }];
+
+  return k8sPatch(model, obj, patch);
+};
+
+export const pauseError = err => {
```

Comment:
I wouldn't pull this error handling in a utility. There's nothing specific about the pause action in this implementation. Instead, I'd use `errorModal` directly in each component.